coupon/src/main/resources/properties/monitoring.yml

```
@@ -47,7 +47,7 @@ spring:
 management:
   server:
     port: 19044
-    address: 172.31.38.190
+#    address: 172.31.38.190
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)